server/sonar-web/src/main/js/app/components/nav/component/ComponentNavMenu.js

```diff
@@ -303,7 +303,9 @@ export default class ComponentNavMenu extends React.PureComponent {
   }
 
   renderExtension = ({ key, name }, isAdmin = false) => {
-    const pathname = isAdmin ? `/project/admin/extension/${key}` : `/project/extension/${key}`;
+    const pathname = isAdmin === true
+      ? `/project/admin/extension/${key}`
+      : `/project/extension/${key}`;
```

Comment:
IMO the code is misleading, I'd better change `withoutGovernance.map(this.renderExtension)` to explicitly pass `isAdmin`